model/Image.php

```diff
@@ -1051,3 +1051,14 @@ public function write($showDebug = false, $forceInsert = false, $forceWrite = fa
 		throw new Exception("{$this->ClassName} can not be written back to the database.");
 	}
 }
+
```

Comment:
Isn't there any kind of helper file where global methods can be defined? It would make some sense because `base64url_decode` and `base64url_decode` are often used as names for these purposes in [other languages](http://scriptogr.am/hdknr/post/base64url-encoding) as well. Granted, I had to change it a little from the usual way, to include `~` instead of `-`... 

If they go into Convert.php, should they be called `raw2base64url` (to be consistent with several other methods there) or just the same names as below?